authors/apps/comments/views.py

```diff
@@ -86,32 +83,34 @@ def get(self, request, pk):
         serialized_data = self.serializer_class(self.comment)
         return Response({"comment": serialized_data.data,
                          "message": (
-                    "Successfully returned details of comment: {}".format(pk)
-                         )},
-                        status=status.HTTP_200_OK)
+                             "Successfully returned details of comment: {}".
+                             format(pk)
+                         )}, status=status.HTTP_200_OK)
 
     def patch(self, request, pk):
         self.get_required_objects(request, pk)
         self.get_author_profile(request)
         if not self.is_author:
             return Response({
-                    "response": "You can only edit a comment you authored",
+                "response": "You can only edit a comment you authored",
```

Comment:
You have changed the key from message to response, is there a particular reason why?

Reply:
Let me work on that. Thanks for the feedback.